src/noise_interfaces/noise_process_interface.jl

```diff
@@ -350,7 +350,11 @@ end
     end
     return out1,out2
   else # Bridge
-    i = searchsortedfirst(W.t,t)
+    if t isa Union{Rational,Integer}
+      i = searchsortedfirst(W.t,t)
+    else
+      i = searchsortedfirst(W.t,t-eps(typeof(t)))
```

Comment:
is this robust to errors in the other direction?

Reply:
I think it should be. I added some tests which I hope would capture errors if wouldn't be robust. If we would reverse the source of NoiseWrapper for the reverse process then it would need an extra fix -- but since we keep the source in the forward time and use searchsortedfirst instead of searchsortedlast here, one basically always wants to decrease t slightly.

Reply:
On tests that aren't ranges, this might need to be relaxed more, i.e. 10eps(t) or 100eps(t)

Reply:
oh I think I got it now.